docs/content/guides/dagster/model_selection.mdx

```diff
@@ -0,0 +1,231 @@
+
+Therefore, model selection is inherently _ad hoc_. The candidate space must be narrowed via an external agent’s judgment before a “best” model architecture can be identified quantitatively. Common approaches include grid search ( candidate architectures specified in a fixed set) and random search ( candidate architectures sampled from a fixed distribution). More sophisticated approaches apply Bayesian inference to intelligently search a bounded candidate space.
+
+All of these methods require constraints defined by the user. It is often difficult to intelligently choose such constraints. It helps to be able to examine detailed output from a set of candidate model architectures.
+
+Below we’ll provide a Dagster-based and slightly-modified implementation of the model selection pipeline described in Caleb Neale's excellent tutorial [Hyper-Parameter Tuning and Model Selection, Like a Movie Star](https://towardsdatascience.com/hyper-parameter-tuning-and-model-selection-like-a-movie-star-a884b8ee8d68). We'll explain each step of the pipeline as we go, but Neale's tutorial is recommended reading for deeper understanding.
+
+## Problem
+
+Our problem is to classify the records in the the famous [Iris dataset](https://archive.ics.uci.edu/ml/datasets/iris). This dataset is a single 150-row table, with each record (row) representing a single flower belonging to one of three classes. Each record has five features, including four physical measurements: sepal length, sepal width, petal length, and petal width. The last feature is the plant class, a categorical feature representing one of three kinds of plant class. Here is a sample from the above-linked version of the data (note that the plant class is labeled `target` and represented by an element of set `{0, 1, 2}`):
+
+     sepal length (cm)  sepal width (cm)  petal length (cm)  petal width (cm)  target
+                   5.1               3.5                1.4               0.2       0
+                   4.9               3.0                1.4               0.2       0
+                   4.7               3.2                1.3               0.2       0
+                   4.6               3.1                1.5               0.2       0
+                   5.0               3.6                1.4               0.2       0
+
+The classification task is to predict the `target` using the physical measurements. Because this is a [supervised learning problem](https://machinelearningmastery.com/supervised-and-unsupervised-machine-learning-algorithms/), we have a clear model performance metric: the fraction of records correctly classified by the model. We'll call this number (ranging from 0-1) the _classification accuracy_.
```

Comment:
"we have a clear model performance metric"

Many would argue that accuracy is often the wrong metric for assessing the performance of classifiers.  E.g. https://stats.stackexchange.com/questions/312780/why-is-accuracy-not-the-best-measure-for-assessing-classification-models.

I'd remove the word "clear" and replace with something like "One performance metric we can use is accuracy"